### ISSUE TITLE
how to implement huggingface BERT + CRF layer?

### ISSUE THREAD

**Issue by lkqnaruto** [huggingface/transformers#14453]
Hi

I wonder how to add CRF layer to a pretrained BERT model? Do I have to overwrite the loss function in the trainer? I really need some help on this, thank you.

**Comment by LysandreJik** [comment]
Hello, thanks for opening an issue! We try to keep the github issues for bugs/feature requests.
Could you ask your question on the [forum](https://discuss.huggingface.co) instead?

Thanks!
